models/slot-group.js

```diff
@@ -0,0 +1,24 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+var ObjectId = Schema.Types.ObjectId;
+var Mixed = Schema.Types.Mixed;
+
+var slotGroup = new Schema({
+  name: {
+    type: String,
+    index: {
```

Comment:
I am not sure the syntax is correct. just use {index: true, unique: true}
See https://github.com/Automattic/mongoose/issues/2232